### PR TITLE
[doc] Improve Pulsar Administration Pulsar dashboard

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Administration Pulsar dashboard document (Administration-dashboard section): http://pulsar.apache.org/docs/en/administration-dashboard/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
@Anonymitaet please review it, thank you~

File: site2/docs/administration-dashboard.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The Pulsar dashboard is a web application that enables users to monitor current
 
 The dashboard is a data collector that polls stats from all the brokers in a Pulsar instance (across multiple clusters) and stores all the information in a [PostgreSQL](https://www.postgresql.org/) database.
 
-A [Django](https://www.djangoproject.com) web app is used to render the collected data.
+You can use a [Django](https://www.djangoproject.com) web app to render the collected data.
```
Comment (Anonymitaet) [inline site2/docs/administration-dashboard.md]:
```suggestion
You can use the [Django](https://www.djangoproject.com) web APP to render the collected data.
```

File: site2/docs/administration-dashboard.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ $ docker run -p 80:80 \
   apachepulsar/pulsar-dashboard:{{pulsar:version}}
 ```
 
-The {@inject: github:`Dockerfile`:/dashboard/Dockerfile} can be found in `dashboard` directory, you can build a image from scratch also:
+You can find the {@inject: github:`Dockerfile`:/dashboard/Dockerfile} in `dashboard` directory, you can build a image from scratch also:
```
Comment (Anonymitaet) [inline site2/docs/administration-dashboard.md]:
```suggestion
You can find the {@inject: github:`Dockerfile`:/dashboard/Dockerfile} in the `dashboard` directory and build an image from scratch as well:
```

**Comment by Jennifer88huang** [comment]
run cpp tests
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run cpp tests
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests

**Pull request merged**